### DISCUSSION TITLE
Can someone provide more instructions on how to use this?

### DISCUSSION THREAD

**Discussion by ghost** [bitbybyte/fantiadl#46] [Q&A] [answered]
cannot figure anything out

**Comment by bitbybyte** [comment, marked as answer]
- Login to Fantia
- Get your session cookie value as explained in the README: https://github.com/bitbybyte/fantiadl#about-session-cookies
- Download latest .exe file from releases: https://github.com/bitbybyte/fantiadl/releases/
- Find where you saved the file, hold Shift + right click
- Select Open PowerShell window here

Now you can type commnands to interact with the program. For example:

- Download a single post or fanclub
 `fantiadl_v1.7.exe --cookie a1b2c3d4 https://fantia.jp/posts/12345`

- Download all followed fanclubs
`fantiadl_v1.7.exe --cookie a1b2c3d4 -f`

Other options are explained in the README.

**Reply by meganeeman** [reply]
Im new here, and try your explanation. But my powershell always said command for fantiadl_v1.7.exe was not found. What should i do first sir?

**Reply by SilentSixx** [reply]
I've done all of this still not working, I tried dragging the fantiadl.exe into the open Powershell and that seemed to work but the cookie and value aren't working now says Unexpected token.

**Reply by meganeeman** [reply]
I can, but it only for free content, cant bypass the paywall tho
